binding/python/porcupine.py

```diff
@@ -39,15 +39,28 @@ class PicovoiceStatuses(Enum):
     class CPorcupine(Structure):
         pass
 
-    def __init__(self, library_path, model_file_path, keyword_file_path, sensitivity):
+    def __init__(
+            self,
+            library_path,
+            model_file_path,
```

Comment:
you can keep the interface the same and in the body of your function have
`if isinstance(model_file_path, list):
     model_file_paths = model_file_path
  else:
    model_file_paths = list(model_file_path)
`

Same for sensitivity.

Reply:
that's an option. I prefer it this way as the argument names go with semantics.